### PR TITLE
update core tensorflow lab to TF 2.0

### PR DESCRIPTION
**Pull request by BenoitDherin**

### CODE REVIEW

File: courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb [hunk 1]
```diff
@@ -4,22 +4,25 @@
    "cell_type": "markdown",
```
Comment (munnm) [inline courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>edit comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (munnm) [inline courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>typo. "Constant constant can not be changed, while variables values can be." to "Constant values can not be changed, while variable values can be."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (munnm) [inline courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>'tf.substrac' to 'tf.subtract'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (munnm) [inline courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>it seems that 'loss' is not created, so throws an error when trying to print the message. I guess here you want to have an extra line with loss = loss_mse(X, Y, w0, w1) after updating gradients?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (munnm) [inline courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>change comment</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (munnm) [inline courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>typo "constant constant"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (BenoitDherin) [reply courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>done</p>

Comment (BenoitDherin) [reply courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>done</p>

Comment (BenoitDherin) [reply courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>done</p>

Comment (BenoitDherin) [reply courses/machine_learning/intro_to_tensorflow/1_core_tensorflow.ipynb]: <p>done</p>
